README.md

```diff
@@ -990,6 +990,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Orah, Psalm Chiemenam](https://github.com/orahpsalm)
   - [Osae-Addo](https://github.com/OsaeAddo)
   - [Osoraa Nwankwo](https://github.com/Osoraa)
+  - [Ojochogwu Dickson](https://github.com/Ojochogwu866)
```

Comment:
Write your name after Ojas to satisfy the alphabetical order requirement. All of you have an O as your first letter, so the ordering starts with the next letter you have which is J, and since Ojas has an A as the third letter, this will make you right after him in the listing.

I hope I made it clear to you, cheers